templates/addons/aurora/mysql-cf.yml

```
@@ -0,0 +1,150 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+  # Customize your Aurora Serverless cluster by setting the default value of the following parameters.
+  {{logicalIDSafe .Name}}DBName:
+    Type: String
+    Description: The name of the initial database to be created in the DB cluster.
+    Default: {{.InitialDBName}}
+    # Cannot have special characters
+    # Naming constraints: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html#RDS_Limits.Constraints
+  {{logicalIDSafe .Name}}DBAutoPauseSeconds:
```

oh I just realized maybe we should replace these parameter and resource names like this:
```suggestion
  {{logicalIDSafe .InitialDBName}}DBAutoPauseSeconds:
```

Because right now if for whatever reason a user wants to initialize multiple Aurora DBs for the same service we will create the same resource names which will clash.

---

Ohh `.Name` is the storage name, not the service name. Since user cannot create multiple storage add-ons with the same name, `.Name` won't clash.

---

On the contrary if they create multiple Aurora storage with the same `InitialDBName` it will clash.